src/EnergyPlus/SolarShading.cc

```diff
@@ -10438,16 +10438,6 @@ void SkyDifSolarShading(EnergyPlusData &state)
             state.dataSurface->Surface(SurfNum).ViewFactorSkyIR *= state.dataSolarShading->SurfDifShdgRatioIsoSkyHRTS(1, 1, SurfNum);
         }
         state.dataSurface->Surface(SurfNum).ViewFactorGroundIR = 1.0 - state.dataSurface->Surface(SurfNum).ViewFactorSkyIR;
```

Comment:
At this point the `state.dataSurface->Surface(SurfNum).ViewFactorSkyIR` is already set due to the change in calling order of surface view factors reset function `InitSurfacePropertyViewFactors(state)` and it is done only once. There is no need to check and update again.

Reply:
The code below is not needed now because the code above already operated on the updated `Surface(SurfNum).ViewFactorSkyIR`.